nadeo_event_api/src/nadeo_event_api/api/structure/settings/script_settings.py

```diff
@@ -2,17 +2,42 @@
 
```

Comment:
Make sure to run `pytest` on this to verify the event payloads are still building correctly